Gemfile

```diff
@@ -75,23 +75,22 @@ gem 'rest-client'
 gem 'noid', '~> 0.8'
 
 group :test do
-  gem "rspec-its"
+  gem "capybara"
+  gem "capybara-select2"
+  gem "database_cleaner"
   gem "ci_reporter_rspec"
+  gem "poltergeist", "~> 1.5"
+  gem "show_me_the_cookies"
   gem "timecop"
```

Comment:
As of Rails 4.1+, timecop isn't really needed (unless your using some it's advance features for accelerating time,etc.). Rails comes with TimeHelpers for mocking out/freezing/jumping with time.

More Info on TimeHelpers: http://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html

So to freeze time and unfreeze time you will just do the following:
```ruby
      before do
        # Freeze time
        travel_to Time.zone.now
      end

      after do
        # Unfreeze time
        travel_back
      end
```